### PR TITLE
MPLSoUDP: fixing Fig1 and adding procedures and controller plane sect…

### PR DESCRIPTION
**Pull request by vb-balazs**
Based on 06.17 discussions. F-lables(s) added to Fig1 and text fixed.
Adding procedures sections. Multiple notes added for clarifications.
Adding controller plane section.

### CODE REVIEW

File: mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml [hunk 1]
```diff
@@ -323,6 +348,53 @@
 
 </section>
 
+<section anchor="ip-flow-id-info"
+                 title="Management and Control Information Summary">
+          <t>
+            The following summarizes the set of information that is needed to
+            identify individual and aggregated DetNet flows:
```
Comment (vb-balazs) [inline mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml:355]: configure DetNet MPLS over IP

File: mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml [hunk 2]
```diff
@@ -323,6 +348,53 @@
...
+              zero (0) effectively means that theses fields are
+              ignored.</t>
+              <t>UDP Source Port. Exact and wildcard matching is
+              required. Port ranges can optionally be used.</t>
+            </list>
+            This information MUST be provisioned per DetNet flow via
+            configuration, e.g., via the controller or management plane.
+          </t>
+          <t>
+            Information identifying a DetNet flow is ordered and
```
Comment (vb-balazs) [inline mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml:382]: No order, because no wildcards, etc.

**Review by louberger — APPROVED** [review]
reviewed in Jun 24 meeting

File: mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml (hunk 1 above)
Comment (vb-balazs) [reply mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml:355]: Add the label (S-label or F-label) to be mapped to UDP/IP
Destination UDP port (full 6-tuple information needed)
No ranges/prefixes/wildcards needed.

**Pull request merged**